### PR TITLE
data:rahul152-ai

### PR DESCRIPTION
**Pull request by rahul152-ai**

### CODE REVIEW

**Review by priyankarpal — APPROVED** [review]
LGTM 👍🏻 Thanks for your contribution

**Review by himanshu1221 — CHANGES_REQUESTED** [review]
Hey your linkedin link doesnt seems like to be working 

`"url": "https://www.linkedin.com/in/ahul-kumar-0674bb214/"`

**Comment by rahul152-ai** [comment]
> Hey your linkedin link doesnt seems like to be working
> 
> `"url": "https://www.linkedin.com/in/ahul-kumar-0674bb214/"`
Thank you for noticing that. I will update that

**Review by priyankarpal — APPROVED** [review]
LGTM 👍🏻

**Review by priyankarpal — DISMISSED** [review]
LGTM

File: data/rahul152-ai.json
Comment (kumarsonsoff3) [inline data/rahul152-ai.json:5]: Did you mean `open source`.
```suggestion
  "bio": "New to open source",
```

**Review by kumarsonsoff3 — COMMENTED** [review]
Rest looks good to me! 👍

**Review by himanshu1221 — APPROVED** [review]
LGTM :+1:  
Thanks for contributing 😃

**Comment by rahul152-ai** [comment]
> LGTM 👍 Thanks for contributing 😃

Who will merges this pull request into [EddieHubCommunity: main]?

**Comment by himanshu1221** [comment]
> > LGTM 👍 Thanks for contributing 😃
> 
> Who will merges this pull request into [EddieHubCommunity: main]?
> 

Maintainer will merge it

**Comment by rahul152-ai** [comment]
@himanshu1221 should I close this pull request or wait to merge it?

**Comment by himanshu1221** [comment]
> @himanshu1221 should I close this pull request or wait to merge it?

You should wait for getting it merge

**Review by kumarsonsoff3 — APPROVED** [review]
Looks good, Thank you for your Contribution! 🎉 

<p align="center"></p>

**Comment by rahul152-ai** [comment]
Hey, @kumarsonsoff3  While setting up this project I got the following error Do I need to install the next-auth/react? 
e5e052f1-1dff-49d1-9ccc-0704a151b205

**Pull request merged**
